lumigator/python/mzai/jobs/eval_lite/eval_lite.py

```diff
@@ -0,0 +1,159 @@
+import json
+from pathlib import Path
+
+import click
+import numpy as np
+import s3fs
+from datasets import load_from_disk
+from loguru import logger
+import evaluate
+
+class EvaluationMetrics:
```

Comment:
@aittalam wdyt about stuffing this all into one script for now? It generally feels weird but it's small enough that I think it's ok since we want to send a minimal amount of files to Ray